endpoint/endpoint_core.c

```diff
@@ -37,10 +37,9 @@
 #define EP_SSL_SEED nonce
 #endif
 
-#define REQ_BODY \
-  "{\"value\": %d, \"message\": \"%s\", \"message_format\": \"%s\", \"tag\": \"%s\", \"address\": \"%s\"}\r\n\r\n"
+#define REQ_BODY "{\"data\":{\"seed\":\"%s\",\"message\":\"%s\"},\"protocol\":\"MAM_V1\"}\r\n\r\n"
```

Comment:
You need to add the "x-api-key" attribute.

Reply:
We need to discuss the way to register on Alfred